app/components/VideoControls.js

```
@@ -0,0 +1,70 @@
+      >
+        <span className="material-icons text-sm">
+          {isMicOn ? 'mic' : 'mic_off'}
+        </span>
+        {isMicOn ? 'Mute' : 'Unmute'}
+      </button>
+      
+      <button 
+        className={`${isCameraOn ? 'bg-blue-500 hover:bg-blue-600' : 'bg-red-500 hover:bg-red-600'} text-white px-4 py-2 rounded transition-colors flex items-center gap-2`}
+        onClick={onToggleCamera}
+        title={isCameraOn ? "Turn off camera" : "Turn on camera"}
+      >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)